src/main/java/com/monitorjbl/plugins/PullRequestListener.java

```diff
@@ -20,9 +20,6 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 
 public class PullRequestListener {
-  private static final String PR_APPROVE_BUCKET = "AUTOMERGE_PR_APPROVAL";
-  private static final String BUILD_APPROVE_BUCKET = "AUTOMERGE_BUILD_APPROVAL";
-  public static final int MAX_COMMITS = 1048576;
```

Comment:
All of these were unused, with the switch to streaming and moving away from the `BucketedExecutor`